src/node/stage/approved.rs

```diff
@@ -642,23 +635,14 @@ impl Approved {
             MessageHash::from_bytes(&bounced_msg_bytes),
             sender,
         );
-
-        // First send parsec gossip to update the peer, then resend the message itself. If the
-        // messages arrive in the same order they were sent, the gossip should update the peer so
-        // they will then be able to handle the resent message. If not, the peer will bounce the
-        // message again.
-
-        self.send_parsec_gossip(core, Some((sender_parsec_version, sender.clone())));
-
-        // With the removal of parsec, sending parsec_gossip won't update the peer anymore.
-        // We will have to send the message later on to allow the peer be updated by other delayed
-        // messages.
-        // TODO: When the testing framework is updated to use real messaging underlayer, i.e. not
-        // using fake clock, the bounced unknown message could be resent immediately. As the network
-        // already took time to exchange messages, which allows the lagging node to be updated.
-        self.gossip_timer_token = Some(core.timer.schedule(self.consensus_engine.gossip_period()));
-        self.bounced_unknown_messages
-            .push((*sender.peer_addr(), bounced_msg_bytes));
+        core.send_direct_message(
```

Comment:
Would be nice to preserve the original comment that was here, just replace "parsec gossip" with "NotifyLagging":

>        // First send parsec gossip to update the peer, then resend the message itself. If the
>        // messages arrive in the same order they were sent, the gossip should update the peer so
>        // they will then be able to handle the resent message. If not, the peer will bounce the
>        // message again.